StripePaymentSheet/StripePaymentSheet/Source/PaymentSheet/PaymentSheet.swift

```diff
@@ -125,6 +125,7 @@ public class PaymentSheet {
             } else {
                 completion(status)
             }
+            self.bottomSheetViewController.contentStack = [self.loadingViewController]
```

Comment:
Feel free to ship as-is, just a suggestion. I think it'd be more obvious that we're starting every `present` call with a loading screen if we did this immediately before presenting on line 190.

Reply:
Unfortunately, this makes the sheet not present at all.

Reply:
! That's surprising, why?

Reply:
I suspect it has something to do with BottomSheetViewController because the viewcontroller is already set to loadingViewController.  I did not investigate it too deeply though.

Reply:
Dug into this a little bit more, and seems that my initial theory around it being the same view is correct.

Option 1:
Ship as-is

Option 2:
In BottomSheetViewController add the following to the contentViewController's didSet:
```
    private var contentViewController: BottomSheetContentViewController {
        didSet(oldContentViewController) {
+            guard self.contentViewController !== oldContentViewController else {
+                return
+            }
``` 
Then as you were suggesting on line 190:
```
        self.bottomSheetViewController.contentStack = [self.loadingViewController]
```

I am kind of leaning towards option 2, it could lead to future footguns in the future.  Thoughts?

Reply:
Ahh that makes perfect sense! Agreed, option 2 is better. Thanks for investigating and making the deeper fix!